packages/twenty-front/src/modules/auth/states/currentWorkspaceState.ts

```diff
@@ -14,6 +14,7 @@ export type CurrentWorkspace = Pick<
   | 'currentBillingSubscription'
   | 'workspaceMembersCount'
   | 'currentCacheVersion'
+  | 'workspaceInvitations'
```

Comment:
It's a proposition to manage invitations though the workspace entity. WDYT @FelixMalfait ?

Reply:
Why not but what's the case for it? This is loaded very often while today we only display invitations in 1 place in settings which clearly isn't access as often. If we were to display them in the main app I'd also have argued it's nice to have them here, but now I'm not convinced